great_expectations/checkpoint/checkpoint.py

```diff
@@ -109,6 +109,10 @@ def run(
             run_id and run_time
         ), "Please provide either a run_id or run_name and/or run_time."
 
+        # If no validations are provided, the combination of expectation_suite_name, batch_request,
+        # and action_list are considered the "default" validation.
+        using_default_validation = not self.validations and not validations
```

Comment:
This currently accounts for runtime behavior but we will definitely need to flesh out our error checking to confidently say this works with runtime args.